schemas/json/layout/expression.schema.v1.json

```diff
@@ -190,6 +191,17 @@
       ],
       "additionalItems": false
     },
+    "func-roles": {
+      "title": "Roles lookup function",
+      "description": "This function will look up a role that the user has for their current party",
+      "type": "string",
+      "items": [
+        { "const":  "externalApi" },
+        { "$ref": "#/definitions/string" },
+        { "$ref": "#/definitions/string" }
```

Comment:
This line should probably be removed. According to the later definition in `expression-functions.ts` this function only accepts one argument, not two.